### DISCUSSION TITLE
Applying Robin boundary condition

### DISCUSSION THREAD

**Discussion by legofernando1999** [usnistgov/fipy#929] [Q&A]
Hi, I'll be brief, I'm trying to solve the following PDE:

<img width="379" alt="image" src="https://github.com/usnistgov/fipy/assets/82280950/c88b787a-5a4d-49af-847d-03a6bbf12aac">

I figured out how to write it all in FiPy notation except for the last boundary condition (The Robin BC). Can someone please help me out with this? I read the documentation already but am not sure how to do it in this particular case. 

Thank you all.

**Comment by guyer** [comment]
Have you read the [documentation on applying Robin conditions](https://www.ctcms.nist.gov/fipy/documentation/USAGE.html#applying-robin-boundary-conditions)? If so, what have you tried? What isn't working for you?
